contracts/events/src/contract.rs

```diff
@@ -52,3 +83,96 @@ fn handle_attributes(deps: DepsMut, attributes: Vec<Attribute>) -> Result<Respon
     deps.api.add_attributes(&attributes)?;
     Ok(Response::default())
 }
+
+fn handle_event_dyn(
```

Comment:
I would like to double check, am I correct in understanding that this function will create 3 events?

Reply:
Yes, it issues the given event three times in different ways. This is because it is a function for tests.

Reply:
understood, thank you.